includes/integration/class-pixelfed.php

```diff
@@ -19,10 +19,23 @@
  */
 class Pixelfed {
 	public function __construct() {
+		add_filter( 'mastodon_api_nodeinfo', array( $this, 'mastodon_api_pixelfed_nodeinfo_software_root' ), 99, 2 );
```

Comment:
Please also add a filter for when the nodeinfo plugin is installed:

```suggestion
		add_filter( 'nodeinfo2_data', array( $this, 'mastodon_api_pixelfed_nodeinfo_software_root' ), 99, 2 );
		add_filter( 'mastodon_api_nodeinfo', array( $this, 'mastodon_api_pixelfed_nodeinfo_software_root' ), 99, 2 );
```